landlab/components/uniform_precip/generate_uniform_precip.py

```diff
@@ -13,7 +13,8 @@
 
 import random
 import numpy as np
-from landlab import Component
+from landlab import Component, ModelGrid
+from six import next
 
 
```

Comment:
No worries... (we are on a multi-part holiday... first the eclipse... now canoeing!)

I'll put it on the call to-do list.